src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -777,6 +778,10 @@ internal void SetPackageState(PackageDownloadHandle packageDownloadHandle, strin
             Package dynPkg;
             if (packageDownloadHandle.Extract(DynamoViewModel.Model, downloadPath, out dynPkg))
             {
+                if (DebugModes.IsEnabled("DynamoPackageStates"))
+                {
+                    dynPkg.PackageState = Package.PackageStates.Loaded;
```

Comment:
Shouldn't we set this only after the call to `LoadPackages`?

Reply:
hmm...I did not attempt to properly set all the states. 
I only focused on the UI/UX side of it.

It may make sense to actually set the states in the `PackageManagerExtension.PackageLoader` ...since we may have more visibility into error states there. Or maybe some other notification system...Not sure yet

I can add a comment that this specific code is temporary...and should be moved somewhere else...And I can move it below as well